docs/examples-profiling-guest.md

```diff
@@ -0,0 +1,14 @@
+# Using Wasmtime's cross-platform profiler
+
+The guest profiling strategy enables in-process sampling and will write the
+captured profile to a file which can be viewed at
+<https://profiler.firefox.com/>.
+
+To use this profiler with the Wasmtime CLI, pass the
+`--profile=guest[,path[,interval]]` flag.
+
+- `path` is where to write the profile, `wasmtime-guest-profile.json` by default
+- `interval` is the duration between samples, 10ms by default
+
+When used with `--wasm-timeout`, the timeout will be rounded up to the nearest
+multiple of the profiling interval.
```

Comment:
This should maybe mention that you can load the profile in perf.firefox.com? Would be nice to include a screenshot as well.

Reply:
It definitely should mention that. Also, it does mention it, on line 5. :grin:

A screenshot or two is a good idea but I'm not feeling up to doing that right this moment, so I'd like to merge without it.

Reply:
Lol whoops don’t know how I missed that